src/main/java/io/jenkins/plugins/synopsys/security/scan/service/scm/SCMRepositoryService.java

```diff
@@ -33,6 +34,12 @@ public Object fetchSCMRepositoryDetails(
                 ? Integer.parseInt(envVars.get(ApplicationConstants.ENV_CHANGE_ID_KEY))
                 : null;
 
+        LoggerWrapper logger = new LoggerWrapper(listener);
+        if (isFixPrOrPrComment && projectRepositoryPullNumber == null) {
+            logger.warn("PRComment is set true but this is not a pull request scan");
```

Comment:
Update the warning messages uniformly with other integrations 

**PR comment is ignored for non PR/MR scan**